pages/test_analytics/flaky_test_assignment.md

```diff
@@ -11,15 +11,21 @@ To enable assignments, you must have at least one team that has access to your s
 
 ## Assigning a test
 
-When viewing your flaky tests, you should now see a list of teams with suite access permissions listed inside the assignment dropdown. From here you may assign, reassign or remove the assignment.
+When viewing your flaky tests, you should now see a list of teams with suite access permissions listed inside the flaky management dropdown. From here you may assign, reassign or remove the assignment.
 
 > 🚧 Assignment permissions
 > All team members have the ability to create, update or remove an assignment. This feature is not restricted to admins.
 
-Tests that are assigned to a team will be updated to display a badge indicating as such. This badge is also visible on issues in the run page.
+Tests that are assigned to a team will be updated to display a badge indicating as such.
 
 <%= image "flaky-test-teams.png", width: 1960/2, height: 630/2, alt: "Flaky test page showing team assignments" %>
 
+## Resolving a flaky test
+
+From the flaky management dropdown, you can resolve your flaky test. Resolving a flaky test will remove the test from **My Assignments** and display a **Marked as resolved** badge within the flaky index. The test will stay in the flaky index until it has not flaked within 7 days. If the test flakes again, it will be considered a **Reoccurring flaky** and will receive a corresponding badge in the index.
```

Comment:
What is the 'flaky management dropdown' as it is verbatim in the UI?

This should be indicated verbatim and in bold text as per our [writing styleguide](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#ui-elements).

Reply:
In the UI it has a label of **Manage flaky test** for the dropdown so I can update the reference here. 
The UI also does use a **Reoccurring Flaky** badge when a flaky reoccurs.

> Furthermore, there does not need to be a screenshot added for this particular UI element.

Did you want me remove the screenshot below for this feature?

Reply:
That would be fantastic @meghan-kradolfer and thanks for confirming the rest.

Happy for the screenshot to stay if it's relevant, bearing in mind that they can add to a slight maintenance overhead in docs when UIs rapidly change over time.

Reply:
The screenshot is only showing the badges that resolution adds, the actual **Resolve** button is shown on other screenshots. I'm happy to remove this one below as it doesn't feel super necessary and keen to make our docs easier to maintain with the rapidly changing UI

Reply:
Thanks for implementing this feedback so quickly!